src/backend/joanie/core/serializers/client.py

```diff
@@ -1194,7 +1194,7 @@ class Meta:
             "enrollment",
             "id",
             "main_invoice_reference",
-            "order_group_id",
```

Comment:
To keep the convention to suffix with `id the property name when this one represents the resource through an id what do you think to call this property `order_group_ids` ?